shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/database/metadata/dialect/OracleDataSourceMetaData.java

```diff
@@ -33,24 +33,39 @@
     
     private static final int DEFAULT_PORT = 1521;
     
-    private final String hostName;
+    private String hostName;
     
-    private final int port;
+    private int port;
     
-    private final String catalog;
+    private String catalog;
     
-    private final String schema;
```

Comment:
Why did you remove the `final`?

Reply:
it is used in method matchWithConnectDescriptorPattern, which is not in constructor method

Reply:
done